.github/workflows/upgrader.base.yml

```
@@ -64,5 +64,5 @@ jobs:
         with:
           username: jrbeverly
           title: "Bump base image to ${{ steps.tag.outputs.image-tag }} tag"
```

Title should follow this format:

```suggestion
          title: "deps(ubuntu): Bump base image to ${{ steps.tag.outputs.image-tag }} tag"
```